### PR TITLE
GAWB-991 fix typo 

### PR DESCRIPTION
**Pull request by jlmwhitt**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  - Acceptance criteria exists and is met
  - Note any changes to implementation from the description
  - Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  - Authentication
  - Authorization
  - Encryption
  - Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- Review cycle:
  - LR reviews
  - Rest of team may comment on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH
  - Submitter updates documentation as needed
  - Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jlmwhitt** [comment]
Updated so that the links open in a new window

**Comment by davidangb** [comment]
Consider using the `GCSFilePreviewLink` component, instead of making direct links to objects in GCS.

We've had lots of problems with direct links, in the case where the user has multiple Google identities in the same browser profile. For instance, see GAWB-359. The `GCSFilePreviewLink` component handles those cases. It also imposes specific UX ... while it'll be consistent with other gs:// links, it'll be different than what you've got now.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs [hunk 1]
```diff
@@ -73,6 +73,8 @@
                          workflow-name (first input-names)
                          workflowId (workflow "workflowId")]
                    (style/create-link {:text workflowId
+                                       :target "_blank"
+                                       :style {:color "-webkit-link" :textDecoration "underline"}
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs:77]: This style is consistent-enough that it should be factored out so we're not violating DRY. Maybe we need a `style/create-external-link` that applies this style by default? No need to fix for this PR but worth following up with another.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs [hunk 2]
```diff
@@ -169,8 +171,12 @@
              [:div {} (submission "submitter")]
              (let [m (js/moment (submission "submissionDate"))]
                [:div {} (.format m "LLL") " (" (.fromNow m) ")"]))
-           (style/create-section-header "Submission ID")
-           (style/create-paragraph (submission "submissionId"))]
+                (style/create-section-header "Submission ID")
+                (style/create-link {:text (style/create-paragraph (submission "submissionId"))
+                               :target "_blank"
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs:176]: Formatting here is weird. Let's tweak your editor.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs [hunk 3]
```diff
@@ -15,19 +15,18 @@
 
 (defn- create-field [label & contents]
   [:div {:style {:paddingBottom "0.25em"}}
-   [:div {:style {:display "inline-block" :width 100}} (str label ":")]
+   [:div {:style {:display "inline-block" :width 130}} (str label ":")]
    contents])
 
 
-(defn- display-value [gcs-uri]
+(defn- display-value [gcs-uri link-label]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs]: Created a PR to suggest a change to this.

**Review by dmohs — APPROVED** [review]
👍

**Pull request merged**